src/components/Fruit.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+
+function Fruit ({fruitName, fruitEmoji, counter, setCounter}) {
+    let setClassName = ""
```

Comment:
De naam van deze variable suggereert dat het een functie is. Ik zou hem daarom gewoon `className` noemen